### ISSUE TITLE
0.9.0 doesn't load

### ISSUE THREAD

**Issue by nixxo** [daveleroy/sublime_debugger#200]
The new updated version generates an error for me and it does not load properly.

0.8.4 is ok, I downgraded and disabled updates for now.

```
startup, version: 4143 linux x64 channel: stable
executable: /opt/sublime_text/sublime_text
application: /opt/sublime_text
working dir: /home/utente
packages path: /home/utente/.config/sublime-text/Packages
state path: /home/utente/.config/sublime-text/Local
zip path: /opt/sublime_text/Packages
zip path: /home/utente/.config/sublime-text/Installed Packages
ignored_packages: ["Vintage"]
Unable to parse color value var(background) at Packages/Debugger/contributes/ColorScheme/Mariana.sublime-color-scheme:5:18
pre session restore time: 0.298097
startup time: 0.426529
git: located Sublime Merge installed at /opt/sublime_merge
git: using configuration from system git install
git: tracking working dir /mnt/748E74598E7415B8/Users/Utente/Progetti/yt-dlp
first paint time: 0.560122
reloading plugin Default.arithmetic
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.colors
reloading plugin Default.comment
reloading plugin Default.convert_color_scheme
reloading plugin Default.convert_syntax
reloading python 3.3 plugin 0_package_control_loader.00-package_control
reloading plugin Default.copy_path
reloading plugin Default.echo
reloading plugin Default.exec
reloading python 3.3 plugin 0_package_control_loader.40-ptyprocess
reloading python 3.3 plugin 0_package_control_loader.55-jsonschema
reloading python 3.3 plugin Package Control.1_reloader
reloading python 3.3 plugin Package Control.2_bootstrap
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.html_print
reloading plugin Default.indentation
reloading plugin Default.install_package_control
reloading plugin Default.keymap
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.rename
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin ShellScript.ShellScript
reloading plugin Debugger.start
reloading python 3.3 plugin Package Control.Package Control
Traceback (most recent call last):
  File "/opt/sublime_text/Lib/python38/sublime_plugin.py", line 325, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.8/importlib/__init__.py", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 808, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/utente/.config/sublime-text/Packages/Debugger/start.py", line 39, in <module>
    debugger33_path = os.path.join(sublime.packages_path(), "Debugger33")
NameError: name 'os' is not defined
reloading python 3.3 plugin SublimeLinter-flake8.linter
reloading python 3.3 plugin SublimeLinter.__init__
reloading python 3.3 plugin SublimeLinter._init
reloading python 3.3 plugin SublimeLinter.active_linters_view
reloading python 3.3 plugin SublimeLinter.busy_indicator_view
reloading python 3.3 plugin SublimeLinter.goto_commands
reloading python 3.3 plugin SublimeLinter.highlight_view
reloading python 3.3 plugin SublimeLinter.log_handler
reloading python 3.3 plugin SublimeLinter.message_view
reloading python 3.3 plugin SublimeLinter.panel_view
reloading python 3.3 plugin SublimeLinter.quick_actions
reloading python 3.3 plugin SublimeLinter.status_bar_view
reloading python 3.3 plugin SublimeLinter.sublime_linter
reloading python 3.3 plugin Terminal.Terminal
plugins loaded
Package Control: Skipping automatic upgrade, last run at 2023-01-15 21:13:47, next run at 2023-01-15 22:13:47 or after

```

**Comment by daveleroy** [comment]
Should be fixed in 0.9.1

**Comment by nixxo** [comment]
Thanks.
